test/loss.jl

```diff
@@ -28,5 +28,7 @@ include("header.jl")
         @test isapprox(nll(k, indices, dims=1), nll(a, indices, dims=1))
         @test isapprox(nll(k, indices, dims=2), nll(a, indices, dims=2))
     end
+    logistic(a[:],a[:])
+    bce(a[:],a[:])
```

Comment:
this is not a test

Reply:
also, logsoftmax should be tested

Reply:
logsoftmax is jus an alias. This will increase test time. Do we really need it ?